contrib/scalar/t/t9099-scalar.sh

```diff
@@ -10,6 +10,9 @@ PATH=$(pwd)/..:$PATH
 
 . ../../../t/test-lib.sh
 
+GIT_TEST_MAINT_SCHEDULER="crontab:test-tool crontab ../cron.txt"
```

Comment:
In the commit message, maybe include a link to https://github.com/microsoft/git/blob/vfs-2.32.0/.github/workflows/scalar-functional-tests.yml and describe that the .NET functional tests validate creating clones against a real server and then operating on them as an end-to-end scenario test.

Reply:
Those tests also assume the functionality that exists in microsoft/git, so it is not appropriate to include such a workflow directly into git.git.

Reply:
👍